docs/code-search/code-navigation/how-to/adding_lsif_to_workflows.mdx

```diff
@@ -40,21 +21,21 @@ Some indexers will work out of the box by just running them in your project root
 ```yaml
 jobs:
   scip-go:
-    # this line will prevent forks of this repo from uploading lsif indexes
+    #this line will prevent forks of this repo from uploading lsif indexes
```

Comment:
I see there are a bunch of comments where the space between the `#` and the first letter is being removed. Is this due to some incorrect formatter setting? Visually, it looks quite odd, because conventionally there is always a space after the comment prefix `#`.